.github/workflows/build-docker-image-rosetta.yaml

```diff
@@ -0,0 +1,72 @@
+name: Upload Binaries and Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Github tag to release binaries for (reusing an existing tag will make the pipeline fail)
+        required: true
+        default: latest
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
```

Comment:
do we still need matrix os ?

Reply:
I did not change it, but there is only one os/arch in the matrix. It should be good. But for clarity we can change.